test/e2e/support/setup-test-framework.js

```diff
@@ -1,23 +1,30 @@
 /**
  * External dependencies
  */
-import puppeteer from 'puppeteer';
+import 'expect-puppeteer';
 
 /**
- * Node dependencies
+ * Internal dependencies
  */
-import { visitAdmin } from './utils';
+import {
+	clearLocalStorage,
+	disablePageDialogAccept,
+	enablePageDialogAccept,
+	setViewport,
+	visitAdmin,
+} from './utils';
 
-const { PUPPETEER_HEADLESS, PUPPETEER_SLOWMO, PUPPETEER_TIMEOUT } = process.env;
+/**
+ * Environment variables
+ */
+const { PUPPETEER_TIMEOUT } = process.env;
 
 // The Jest timeout is increased because these tests are a bit slow
 jest.setTimeout( PUPPETEER_TIMEOUT || 100000 );
 
 beforeAll( async () => {
-	global.browser = await puppeteer.launch( {
-		headless: PUPPETEER_HEADLESS !== 'false',
-		slowMo: parseInt( PUPPETEER_SLOWMO, 10 ) || 0,
-	} );
+	await setViewport( 'large' );
```

Comment:
Should we set this `beforeEach` ? Seems like it might otherwise be prone to leakage between tests (particularly looking at `sidebar-behaviour.test.js`).

Reply:
Yes, we can.